codebuild/py312/decrypt_hkeyring_with_net.yml

```diff
@@ -0,0 +1,34 @@
+version: 0.2
+
+env:
+  variables:
+    TOXENV: "py312-full_decrypt-mpl"
+    AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_KEY_ID: >-
+      arn:aws:kms:us-west-2:658956600833:key/b35311ef1-d8dc-4780-9f5a-55776cbb2f7f
+    AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_KEY_ID_2: >-
+      arn:aws:kms:eu-central-1:658956600833:key/75414c93-5285-4b57-99c9-30c1cf0a22c2
+    AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_MRK_KEY_ID_1: >-
+      arn:aws:kms:us-west-2:658956600833:key/mrk-80bd8ecdcd4342aebd84b7dc9da498a7
+    AWS_ENCRYPTION_SDK_PYTHON_INTEGRATION_TEST_AWS_KMS_MRK_KEY_ID_2: >-
+      arn:aws:kms:us-east-1:658956600833:key/mrk-80bd8ecdcd4342aebd84b7dc9da498a7
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.12
+  pre_build:
+    commands:
+      # Download previously generated vectors
+      - aws s3 cp s3://generated-vectors-artifacts-bucket/$CODEBUILD_RESOLVED_SOURCE_VERSION/312_hkeyring_manifest.zip 312_hkeyring_manifest.zip
+      - unzip 312_hkeyring_manifest.zip
+
+      # Download published .NET ESDK so we don't have to build from source
+      - dotnet add NETTestVectors package AWS.Cryptography.EncryptionSDK --version 4.0.1
+
+      # Clone SDK-Dafny repo to get test vectors runner
+      - git clone git@github.com:aws/aws-encryption-sdk-dafny.git
+      - cd aws-encryption-sdk-dafny/AwsEncryptionSDK/runtimes/net/TestVectorsNative/TestVectors
+      - export DAFNY_AWS_ESDK_TEST_VECTOR_MANIFEST_PATH=../../../../../../312_hkeyring_manifest/manifest.json
+  build:
+    commands:
+      - dotnet test --framework net48
```

Comment:
Issue: I thought net48 is only supported on Windows.
@josecorella 
You probably want to use net6.0.

Reply:
net48 is meant to run only on windows. It may succeed on a different OS but it is not guaranteed.

Reply:
Yep, this is WIP. I only opened a draft PR so the code is "somewhere" and has "something" running in codebuild CI.